source/game_screen.py

```diff
@@ -3,72 +3,85 @@
 from kivy.graphics import *
 from kivy.uix.screenmanager import Screen
 
-from ball import Ball
 from camera import Camera
-from physics_engine import PhysicsEngine
 from vector import Vec2D
+from ball import Ball
+import physics_engine
+import player
 
+class GameData:
+    main_camera = Camera(pos=Vec2D(0.0, 0.0), size=20.0)
+    player = None
+    balls = []
 
 class GameScreen(Screen):
     main_camera = Camera(pos=Vec2D(0.0, 0.0), size=7.0)
```

Comment:
Why main_camera is defined in GameScreen and also GameData? I think it should be only in GameData